### PR TITLE
docs(collaboration): add documentation about necessary email address

### PR DESCRIPTION
**Pull request by vpellegrino**
## What is the purpose of the change

Update the documentation on docs.camunda.io to reflect that the Web Modeler expects users to have email addresses assigned.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/collaboration.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ There are four roles with different levels of access rights that can be assigned
 
 ### Inviting users to projects
 
+:::note
+Web Modeler expects users to have email addresses to receive invitations correctly.
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/collaboration.md:32]: You originally said "expects to have email addresses **assigned**" here -- do you mean they just need an email address, or an email address assigned to them within Modeler?

**Review by christinaausley — COMMENTED** [review]
Should this also be applied in the current version of docs? If so, they changes should be copied into `8.1`. Let me know if I can assist with this 👍

File: docs/components/modeler/web-modeler/collaboration.md (hunk 1 above)
Comment (vpellegrino) [reply docs/components/modeler/web-modeler/collaboration.md:32]: @christinaausley, thanks for your question!
They need an email address associated with their account in the identity management system (for instance, Keyclock in the self-managed context). They can join the current project with the invitation, so the Modeler itself. To receive an invitation, you'll need an email address.
Please let me know if what I wrote makes sense and if you need further clarification. Feel free to propose a more descriptive note here in the doc.

**Comment by vpellegrino** [comment]
> Should this also be applied in the current version of docs? If so, the changes should be copied into `8.1`. Let me know if I can assist with this 👍

@christinaausley I discussed this point yesterday, and AFAIK, this change will be part of the next release, so there is no need to backport it to 8.1
@CatalinaMoisuc, please, correct me if I'm wrong.

File: docs/components/modeler/web-modeler/collaboration.md (hunk 1 above)
Comment (vpellegrino) [reply docs/components/modeler/web-modeler/collaboration.md]: @christinaausley done with https://github.com/camunda/camunda-platform-docs/pull/1793/commits/ed1e69359fccd3b276d293a2737ba0ca142bf0a1

**Review by christinaausley — APPROVED** [review]

**Pull request merged**